docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.8"
```

Comment:
should be 3.9 - adjusted in my branch

Reply:
I don't see a `3.9` in the [compose file formats](https://docs.docker.com/compose/compose-file/#compose-and-docker-compatibility-matrix) unless I am missing something.

Reply:
you = right, me = wrong